### PR TITLE
Add real time feed api sample code

### PR DESCRIPTION
**Pull request by cwxie-google**
We're having 5 new apis and I'm adding php sample code for those new five apis.

### CODE REVIEW

**Comment by cwxie-google** [comment]
@bshaffer Hi Brent, could you please take a look at this PR or assign to the right one to review it?
Thanks

File: asset/composer.json [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+        "google/cloud-pubsub": "^1.11.1",
+        "google/cloud-asset": "^0.5.0",
         "symfony/console": " ^3.0"
     },
     "autoload": {
         "files": [
             "src/export_assets.php",
-            "src/batch_get_assets_history.php"
+            "src/batch_get_assets_history.php",
+	        "src/create_feed.php",
```
Comment (bshaffer) [inline asset/composer.json]: nit: whitespace here

File: asset/src/create_feed.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+function create_feed($parent, $feedId, $topic, $assetNames)
+{
+    $client = new AssetServiceClient();
+    
+    $feed = new Feed();
+    $feedOutputConfig = new FeedOutputConfig();
+    $pubsubDestination = new PubsubDestination();
+    $pubsubDestination->setTopic($topic);
+    $feedOutputConfig->setPubsubDestination($pubsubDestination);
```
Comment (bshaffer) [inline asset/src/create_feed.php]: nit, but let's have this ordered better, like this:

```php
    $pubsubDestination = (new PubsubDestination())
        ->setTopic($topic);

    $feedOutputConfig = (new FeedOutputConfig())
        ->setPubsubDestination($pubsubDestination);

    $feed = (new Feed())
        ->setAssetNames($assetNames)
        ->setFeedOutputConfig($feedOutputConfig);
```

File: asset/src/delete_feed.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+use Google\Cloud\Asset\V1\AssetServiceClient;
+
+/**
+ * Get real time feed given its name.
+ */
+function delete_feed($feedName)
+{
+    $client = new AssetServiceClient();
+
+    $client->DeleteFeed($feedName);
```
Comment (bshaffer) [inline asset/src/delete_feed.php]: casing should be `deleteFeed`

File: asset/src/create_feed.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    $feed->setAssetNames($assetNames);
+    $feed->setFeedOutputConfig($feedOutputConfig);
+
+    $createdFeed = $client->CreateFeed($parent, $feedId, $feed);
```
Comment (bshaffer) [inline asset/src/create_feed.php]: casing should be `createFeed`

File: asset/src/get_feed.php [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+use Google\Cloud\Asset\V1\AssetServiceClient;
+
+/**
+ * Get real time feed given its name.
+ */
+function get_feed($feedName)
+{
+    $client = new AssetServiceClient();
+
+    $feed = $client->GetFeed($feedName);
```
Comment (bshaffer) [inline asset/src/get_feed.php]: casing should  be `getFeed`

File: asset/src/get_feed.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+    echo 'Feed ' . $feed->getName() . ' got.';
```
Comment (bshaffer) [inline asset/src/get_feed.php]: This does not read well. Should be:

```php
printf('Got feed "%s".', $feed->getName());
```

File: asset/src/list_feeds.php [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+use Google\Cloud\Asset\V1\AssetServiceClient;
+
+/**
+ * List real time feeds sgiven their parents.
+ */
+function list_feeds($parent)
+{
+    $client = new AssetServiceClient();
+
+    $feed = $client->ListFeeds($parent);
```
Comment (bshaffer) [inline asset/src/list_feeds.php]: casing should be `listFeeds`

File: asset/src/list_feeds.php [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+    echo 'Feeds listed';
```
Comment (bshaffer) [inline asset/src/list_feeds.php]: No feeds were actually listed, would probably make more sense like this:

```php
$feeds = $client->listFeeds($parent);

printf('Feeds for %s:' . PHP_EOL, $parent);
foreach ($feeds as $feed) {
    print($feed->getName() . PHP_EOL);
}
```

File: asset/src/update_feed.php [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+{
+    $client = new AssetServiceClient();
+    
+    $new_feed = new Feed();
+    $new_feed->setName($feedName);
+    $new_feed->setAssetNames($assetNames);
+    $updateMask = new FieldMask();
+    $updateMask->setPaths(['asset_names']);
+    
+    $updated_feed = $client->UpdateFeed($new_feed, $updateMask);
```
Comment (bshaffer) [inline asset/src/update_feed.php]: casing should be `updateFeed` for the method call and `$updatedFeed` for the variable

File: asset/src/update_feed.php [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+use Google\Protobuf\FieldMask;
+
+/**
+ * Create a real time feed.
+ */
+function update_feed($feedName, $assetNames)
+{
+    $client = new AssetServiceClient();
+    
+    $new_feed = new Feed();
```
Comment (bshaffer) [inline asset/src/update_feed.php]: Let's be consistent with our casing and make this variable `$newFeed`

File: asset/src/update_feed.php [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+    $new_feed->setName($feedName);
+    $new_feed->setAssetNames($assetNames);
+    $updateMask = new FieldMask();
+    $updateMask->setPaths(['asset_names']);
+    
+    $updated_feed = $client->UpdateFeed($new_feed, $updateMask);
+
+    echo 'Feed Updated ' . $updated_feed;
```
Comment (bshaffer) [inline asset/src/update_feed.php]: Let's make this `'Feed Updated: ' . $updatedFeed . PHP_EOL;`. All `echo/print` statements should have an appended new line.

File: asset/src/update_feed.php [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+ */
+function update_feed($feedName, $assetNames)
+{
+    $client = new AssetServiceClient();
+    
+    $new_feed = new Feed();
+    $new_feed->setName($feedName);
+    $new_feed->setAssetNames($assetNames);
+    $updateMask = new FieldMask();
+    $updateMask->setPaths(['asset_names']);
```
Comment (bshaffer) [inline asset/src/update_feed.php]: Similar issue here, let's format these like follows:

```php
    $newFeed = (new Feed())
        ->setName($feedName)
        ->setAssetNames($assetNames);

    $updateMask = (new FieldMask())
        ->setPaths(['asset_names']);
```

**Comment by meredithslota** [comment]
This PR is quite stale at this point, and the Asset API has autogenerated samples that also cover these use cases here: https://github.com/googleapis/google-cloud-php/tree/main/Asset/samples/V1/AssetServiceClient Closing this accordingly, please reopen if this is still active.

**Pull request closed without merging**